docs/getting-started/products/yvaults/vault-factory.md

```diff
@@ -43,21 +43,36 @@ Factory-deployed yVaults for Curve LP tokens contain up to three ready-made yiel
 
-Now, with factory strategies, the swap logic is decoupled from the strategy and is performed in a separate transaction from the harvest. Swap transactions must remain permissioned in order to stay safe from MEV attacks. Harvests, as before, rebalance debt between strategies and put idle funds to work into strategies, but also pull accumulated rewards into the strategy contract so they can be swapped. But as soon as swaps are complete, anybody can call harvest to recognize the profit that the swap created and airdropped to the strategy. While harvests will be permissionless on this specific subset of vaults, Yearn will continue to use standard keeper automation to call harvests even if nobody else does.
+### Traditional Strategies
 
-Anyone can call harvest via the harvest() function on the keeper found at [keeper.factory.ychad.eth](https://etherscan.io/address/keeper.factory.ychad.eth#writeContract). Simply put in the strategy address from the factory vault which you would like to call harvest on, and click write.
+Traditional strategies have the token swap logic directly embedded in them. As a result, a harvest call simultaneously performs debt rebalances and converts them into realized profit in one transaction.
 
-### Determine Accumulated Rewards
-This section will help you know how much profit has accumulated, but calling harvest will **not** recognize profits unless they are sitting in the strategy's contract address. Also if there are reward tokens in the strategy's contract, those values should be added to your calculation in the section below. 
+### Factory Strategies
+
+Factory strategies separate swap logic from the strategy, executing it in a different transaction from the harvest. This ensures that swap transactions remain permissioned, safeguarding them from Miner Extractable Value (MEV) attacks.
+
+Harvests in factory strategies still rebalance debt between strategies and allocate idle funds to strategies, while also pulling accumulated rewards into the strategy contract to be swapped by the ySwaps team. Once swaps are completed, which may take several days, anyone can call harvest a second time to acknowledge the profit generated by the swap and distribute it to the vault making pricePerShare increase.
+
+Although harvests will be permissionless for this specific group of vaults, Yearn will continue utilizing standard keeper automation to call harvests, even if no one else does. These automated harvests are run by keep3r.network and are often dependent on network congestion and transaction viability. For example, if the network conditions are consistently unfavorable, there may be delays between harvests.
+
+### Calling Harvest
+
+To call harvest, use the harvest() function on the keeper found at keeper.factory.ychad.eth. Simply enter the strategy address from the factory vault you wish to call harvest on and click “write.”
```

Comment:
you deleted the link here [keeper.factory.ychad.eth](https://etherscan.io/address/keeper.factory.ychad.eth#writeContract)

Reply:
```suggestion
To call harvest, use the harvest() function on the keeper found at [keeper.factory.ychad.eth](https://etherscan.io/address/keeper.factory.ychad.eth#writeContract). Simply enter the strategy address from the factory vault you wish to call harvest on and click “write.”
```